app/components/supplementary-file-browser.js

```diff
@@ -6,11 +6,21 @@ export default Ember.Component.extend({
     scrollAnim: '',
     numShowing: 6,
     selectedFile: {},
-
+    showArrows: Ember.computed('numShowing', 'supplementList.[]', function() {
+        return (this.get('numShowing') < this.get('files').length);
+    }),
     files: Ember.computed('fileList', 'fileList.[]', 'primaryFile', function() {
         //Returns the list with primaryFile moved to the front
         let files = this.get('fileList');
 
+        //TODO: Grab get primary file from preprint object
+        if (files) {
+            const fakePrimary = files.get('firstObject');
```

Comment:
Why fake? This is showing actual file in the project. Should we change the variable name?